src/rich-text/plugins/placeholder.ts

```diff
@@ -0,0 +1,40 @@
+import { Fragment } from "prosemirror-model";
+import { Plugin } from "prosemirror-state";
+import { EditorView } from "prosemirror-view";
+
+function showPlaceholder(content: Fragment<any>, textContent: string) {
```

Comment:
If you're using vscode, make sure you install the recommended extensions. The project is already set up in such a way that they automatically appear under the Extensions tab as "recommended". In this case, your Eslint extension would have caught the warning the GH action is throwing.

Reply:
I think we can drop the `any` as the default types already imply it. I'd love to make our ProseMirror types a little more specific, but it's a bit of a pain to do so cleanly.

```suggestion
function showPlaceholder(content: Fragment, textContent: string) {
```